helios-testing/src/main/java/com/spotify/helios/testing/HeliosDeploymentResource.java

```diff
@@ -64,8 +65,8 @@ public void before() throws Throwable {
     Polling.awaitUnchecked(30, TimeUnit.SECONDS, new Callable<Boolean>() {
       @Override
       public Boolean call() throws Exception {
-        final HostAndPort hap = deployment.address();
-        final SocketAddress address = new InetSocketAddress(hap.getHostText(), hap.getPort());
+        final URI uri = Iterables.getOnlyElement(deployment.uris());
```

Comment:
Sorry this code is slightly outdated. Do we want to make the `deployment` attribute a `HeliosDeployment` instance? In that case, there might be more than one URI if it's an `ExistingHeliosDeployment`.